node_cli/utils/validations.py

```diff
@@ -0,0 +1,99 @@
+#   -*- coding: utf-8 -*-
+#
```

Comment:
I think we should split this module into two separate. One for the global config and other for decorators. Also there is `validate` command in cli, and the purpose of current module (or cli command) may be confusing.

Reply:
Split into `global_config` and `decorators`

Reply:
Updated tests